app/login/page.tsx

```diff
@@ -29,55 +29,56 @@ export default function Login() {
   };
 
   return (
-    <div className="h-screen w-full">
```

Comment:
Q: Why did you replace these tailwind styles in favor of the ones you implemented here?

Reply:
I reworked the page from the ground up. When I created it for the MVP, I did a quick and dirty version to get the desktop version created.

Here I coded mobile first and went up from there. As such, the page needed restructuring and different classes.